api/nodes/serializers.py

```diff
@@ -236,28 +239,13 @@ class NodeSerializer(TaxonomizableSerializerMixin, JSONAPISerializer):
         'subjects',
     ])
 
-    non_anonymized_fields = [
-        'id',
-        'title',
-        'description',
-        'category',
-        'date_created',
-        'date_modified',
-        'registration',
-        'tags',
-        'public',
-        'license',
-        'links',
-        'children',
-        'comments',
-        'contributors',
-        'files',
-        'node_links',
-        'parent',
-        'root',
-        'logs',
-        'wikis',
-        'subjects',
+    anonymized_fields = [
+        'current_user_can_comment',
+        'forks',
+        'citation',
+        'custom_citation',
+        'node_license',
+        'registrations',
```

Comment:
From looking at the tests, these are the only fields I could definitely tell we intended to be anonymized. Some of the fields not in `non_anonymized_fields` were probably unintentional.  Should some of the other current_user fields be anonymized? `current_user_permissions` for example?

Reply:
Current user permissions should always be `read` when using a valid VOL. But we'd definitely break the front end if we hid that field.